### PR TITLE
Fix/modify relay comment test

### LINKED ISSUE #304: [Relay][Comment] Modify comment test
**Is your feature request related to a problem? Please describe.**
There're serveral parts can be modified in comment test.
- remove unused imported objects.
- move test content to global variable
- use stringifyBigInts to whole request body
https://github.com/social-tw/social-tw-website/blob/4951dc4803ccb79df7fdb9ea6e83205310b08c5c/packages/relay/test/comment.test.ts#L102-L107
- remove await on line 159
https://github.com/social-tw/social-tw-website/blob/4951dc4803ccb79df7fdb9ea6e83205310b08c5c/packages/relay/test/comment.test.ts#L159

**Describe the solution you'd like**
There's a TODO comment on line 81
https://github.com/social-tw/social-tw-website/blob/4951dc4803ccb79df7fdb9ea6e83205310b08c5c/packages/relay/test/comment.test.ts#L81
if we need to test the edge case for the largest length of the content, then we need to write a method to generate the content randomly, otherwise, it's enough to use one global mock content for test.

### PR DESCRIPTION
**Pull request by FoodChain1028**
## Summary

remove unused import,  remove await, converted test content into global variable, use stringifyBigInts to encapsulate response.

## Linked Issue

close #304 

## Tests

Comment test in `relay`

## Possible Impacts

X

## Verification Steps

if you want just try single test:
```shell
yarn relay test ./test/comment.test.ts
```

For relay test
```shell
yarn relay test
```

## Checklist

* [X] remove unused imports
* [X] remove await for getting attesterId
* [X] move test content to global variable
* [X] use stringifyBigInts to whole request body
* [X] Run `yarn lint:fix`

### CODE REVIEW

File: packages/relay/test/comment.test.ts [hunk 1]
```diff
@@ -3,11 +3,7 @@ import { ethers } from 'hardhat'
 import { io } from 'socket.io-client'
 import { CircuitConfig } from '@unirep/circuits'
```
Comment (chentihe) [inline packages/relay/test/comment.test.ts:4]: remove unused object

**Review by chentihe — APPROVED** [review]
